api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -200,6 +200,11 @@ model ApiProfile {
   /** The internet visibility of the OpenShift API server */
   @visibility(Lifecycle.Read, Lifecycle.Create)
   visibility?: Visibility = Visibility.Public;
+
+  /** The list of authorized IPv4 CIDR blocks allowed to access the API server. Maximum 5000 entries. */
```

Comment:
Here I think we have a lack of definition of constraints around this:
* Are we going to allow overlapping of ranges?
* Are we going to somehow consider the NSG / Subnets related to the Cluster / Node Pools?
* Are we going to prevent from some specific blocks to be provided?

Similarly to other comments, we might not get the needed answers today but we should follow up on this

cc @nimrodshn